# HG changeset patch
# Parent 26489830984dad5307e86004190da76a83ac0a4e
Enhance spi_flash_erase to display current activity.

diff --git a/common/cmd_sf.c b/common/cmd_sf.c
--- a/common/cmd_sf.c
+++ b/common/cmd_sf.c
@@ -139,11 +139,13 @@ static int do_spi_flash_erase(int argc, 
 	if (*argv[2] == 0 || *endp != 0)
 		goto usage;
 
+	puts("Erasing SPI flash...");
 	ret = spi_flash_erase(flash, offset, len);
 	if (ret) {
 		printf("SPI flash %s failed\n", argv[0]);
 		return 1;
 	}
+	puts("Done\n");
 
 	return 0;
 
